dev-3/dev-3/FirstCriterionTeamBuilder.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace dev_3
+{
+    /// <summary>
+    /// This class builds a team by the first criterion (max performance for customers money).
+    /// </summary>
+    class FirstCriterionTeamBuilder : TeamBuilder
+    {
+        public override Team Choose(int customerMoney, int neededPerformance)
+        {
+            if (customerMoney < 500)
+            {
+                throw new FormatException("We cannot create a team without money, the junior's salary is 500!");
+            }
+            Team team = new Team();
+            while (customerMoney > 0)
+            {
+                if (customerMoney >= 1800)
```

Comment:
if you change Lead's salary you will need to go through all code and change this value